app/models/user.rb

```diff
@@ -66,4 +68,19 @@ def delete_identities
       id.destroy
     end
   end
+
+  private
+    # Private: Create an identity for local file handling when user is created.
+    # Future: This should only happen if user has a password.
+    #
+    # Returns nothing
+    def append_local_identity
+      if own_password && !identities.exists?(service: 'local')
+        local_identity = Identity.new(service: 'local',
+                                      token: SecureRandom.hex(5),
+                                      name: 'Bruse',
+                                      uid: SecureRandom.hex(5))
```

Comment:
I'm changing the uid to use `SecureRandom.uuid` instead.

From [Wikipedia](http://en.wikipedia.org/wiki/Universally_unique_identifier):

> The identifier size and generation process need to be selected so as to make this sufficiently improbable in practice. Anyone can create a UUID and use it to identify something with reasonable confidence that the same identifier will never be unintentionally created by anyone to identify something else. Information labeled with UUIDs can therefore be later combined into a single database *without needing to resolve identifier* (ID) conflicts.